spec/features/exhibits/translation_editing_spec.rb

```diff
@@ -295,6 +295,7 @@
 
         visit spotlight.search_exhibit_catalog_path(exhibit, q: '*', locale: 'fr')
 
+        find('button[aria-label="plus »"]').click # Hamburger icon
```

Comment:
was a test failing without this?

Reply:
Yes, because at the default width, the facets are folded up.